_posts/2011-04-13-reference-data-caching-walkthrough.html

```diff
@@ -13,5 +13,5 @@
 ---
 <div>
 <p>Tim Laverty from the DataFX team at Microsoft <a href="http://blogs.msdn.com/b/astoriateam/archive/2011/04/13/reference-data-caching-walkthrough.aspx"> just posted a walk-through</a> showing how to setup reference data caching using the recently released ' <a href="http://www.microsoft.com/downloads/en/details.aspx?FamilyID=eb4501ca-02f3-42a4-838a-229e8c77d1c6&amp;displaylang=en"> Reference Data Caching CTP</a>' (aka the 'Microsoft WCF Data Services For .NET March 2011 Community Technical Preview with Reference Data Caching Extensions').</p>
-<p>Take a <a href="http://blogs.msdn.com/b/astoriateam/archive/2011/04/13/reference-data-caching-walkthrough.aspx"> look</a> and as always let the OData community know what you think on the <a title="Mailing List" href="https://www.odata.org/join-the-odata-discussion/">Mailing List</a>.</p>
+<p>Take a <a href="http://blogs.msdn.com/b/astoriateam/archive/2011/04/13/reference-data-caching-walkthrough.aspx"> look</a> and as always let the OData community know what you think on the <a title="Mailing List" href="/join-the-odata-discussion/">Mailing List</a>.</p>
```

Comment:
Error 403 on accessing http://blogs.msdn.com/b/astoriateam/archive/2011/04/13/reference-data-caching-walkthrough.aspx

Reply:
@KenitoInc Fixing dead links that point to external resources was beyond the scope of this pull request. In any event, I didn't have a valid alternative I could replace them with.